backend/listing/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+from . import views
+
+router = DefaultRouter()
```

Comment:
No need to have two Viewsets that get the same job done, use just one. Also, no need to add to the `listings` name. Having a simpler paths in the API makes it easier to use and understand.

Desired API paths:
`/api/listing`, `/api/listing/1/`

Reply:
@milensski you might have to change your implementation afterwards.

Reply:
- To perform listing search for all users use base endpoint.
- To perform listing search for specific user use endpoint with ?user_id=<user_id> parameter.
- To perform PUT or PATCH use endpoint with /<listing_id> parameter.